Simple 'make test' broken

After compiling deal.ii 8.5 and then ASPECT pre2.0, the quick mpi test 'make test' is unable to be completed. 

When in aspect/build, 'make test' produces: 

"make: *** No rule to make target `test'."

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -164,6 +164,19 @@ IF(EXISTS ${CMAKE_SOURCE_DIR}/tests/CMakeLists.txt)
   IF(NOT test_cmake_result EQUAL 0)
     MESSAGE(FATAL_ERROR "ERROR: Test project could not be configured.")
   ENDIF()
+
+
+  IF(POLICY CMP0037)
+    # allow to override "test" target
+    CMAKE_POLICY(SET CMP0037 OLD)
+  ENDIF()
+
+  ADD_CUSTOM_TARGET(test
+    COMMAND ${CMAKE_COMMAND} --build . --target test
+    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/tests
+    DEPENDS aspect
+    COMMENT "Running tests ...")
+
 ENDIF()
 
 

